react/store-global.css

```diff
@@ -31,43 +63,40 @@ body {
   }
 }
 
-/*
-  Media Query Extensions:
-  -s  = 20em
-  -ns  = 40em
-  -m  = 40em
-  -l  = 64em
-  -xl  = 80em
-*/
+.vtex-store__template > div {
+  min-height: 100vh;
+  display: flex;
+  flex-direction: column;
+}
 
-.vtex-page-padding {
-  padding-left: 0;
-  padding-right: 0;
+.vtex-store__template > div > div:last-child {
+  margin-top: auto;
 }
 
 @media only screen and (min-width: 20em) {
-  .vtex-page-padding {
-    padding-left: .5rem;
-    padding-right: .5rem;
+  body, .vtex-page-pd {
```

Comment:
Why don't use the classes our Tachyons already provide?

The same effect could be achieved by writing:

```
className="ph3 ph0-ns"
```